<?xml version="1.0"?>
<launch>

    <arg name="id_robot" default="robot"/>
	<arg name="node_name" default="save_robot_poi"/>
    <arg name="folder" default="$(find poi_manager)/config"/>
	<arg name="filename" default="test"/>

    <arg name="global_frame" default="robot_map" />
    <arg name="base_frame" default="robot_base_link" />
	<arg name="has_ptz" default="false" />
	<arg name="ptz_frame" default="robot_flir_ptu_5_stand_link" />

	<node pkg="poi_manager" name="$(arg node_name)" type="save_robot_poi.py" output="screen">
		<param name="folder" value="$(arg folder)"/>
		<param name="filename" value="$(arg filename)"/>
		<param name="global_frame" value="$(arg global_frame)" />   
        <param name="base_frame" value="$(arg base_frame)" /> 
		<param name="has_ptz" value="$(arg has_ptz)" /> 
		<param name="ptz_frame" value="$(arg ptz_frame)" /> 
	</node>

</launch>
